# Практическое задание к 6 семинару (Сравнение долей. Построение доверительного интервала)

## Задание 1
```
Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.
```

$\sigma = 16$

$X_{ср} = 80$

$n = 256$

$\alpha = 0.05 --> Z_{\frac{\alpha}{2}} = 1.96$

Доверительный интервал лежит на отрезке $X_{ср} \pm Z_{\frac{\alpha}{2}} * \frac{\sigma}{\sqrt{n}} = 80 \pm 1.96 * \frac{16}{\sqrt{256}} = 80 \pm 1.96$

**Ответ: доверительный интервал находится в промежутке [78.04 : 81.96]**

## Задача 2
```
В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.
```

In [13]:
import numpy as np
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
mean = np.mean(a)
sigma = np.std(a, ddof=1)
print(mean, sigma)

6.590000000000001 0.4508017549014448


$X_{ср} = 6.59$

$\alpha = 0.05 --> Z_{\frac{\alpha}{2}} = 1.96$

$n = 10$

$\sigma = 0.45$

Доверительный интервал лежит на отрезке $X_{ср} \pm Z_{\frac{\alpha}{2}} * \frac{\sigma}{\sqrt{n}} = 6.59 \pm 1.96 * \frac{0.45}{\sqrt{10}} = 80 \pm 0.28$

**Ответ: доверительный интервал находится в промежутке [6.31 : 6.87]**

## Задание 3
```
На препарате А положительный результат лечения наблюдается у 17 из 32 пациентов, а на препарате В у 9 из 22. Построить 95% доверительный интервал для разности долей.
Обнаружены ли статистически значимые различия?
```

$n_1 = 32$

$m_1 = 17$

$n_2 = 22$

$m_2 = 9$

$P_{общ} = \frac{m_1 + m_2}{n_1 + n_2} = \frac{17 + 9}{32 + 22} = 0.48$

$P_1 = \frac{17}{32} = 0.53$

$P_2 = \frac{9}{22} = 0.41$

$\alpha = 0.05 --> Z_{\frac{\alpha}{2}} = 1.96$

$\Delta =  P_1 - P_2 = 0.53 - 0.41 = 0.12$

$S_{\Delta} = \sqrt{P_{общ} * (1 - P_{общ}) * (\frac{1}{n_1} + \frac{1}{n_2})} = \sqrt{0.48 * (1 - 0.48) * (\frac{1}{32} + \frac{1}{22})} = 0.13836$

Доверительный интервал лежит на отрезке $\Delta \pm Z_{\frac{\alpha}{2}} * S_{\Delta} = 0.12 \pm 1.96 * 0.13836 = 0,12 \pm 0.27$

$Z = \frac{\mid P_1 - P_2 \mid - \frac{1}{2}*(\frac{1}{n_1} + \frac{1}{n_2})}{ \sqrt {P_{общ} * (1 - P_{общ}) * (\frac{1}{n_1} + \frac{1}{n_2})}} = \frac{\mid 0.53 - 0.41 \mid - \frac{1}{2}*(\frac{1}{32} + \frac{1}{22})}{ \sqrt {0.48 * (1 - 0.48) * (\frac{1}{32} + \frac{1}{22})}} = 0.59$

**Ответ: доверительный интервал для разности долей равен [-0.15 : 0.39], статистически значимых различий нет, так как $Z_{\frac{\alpha}{2}} > Z$.**